salt/modules/zk_concurrency.py

```diff
@@ -276,6 +277,8 @@ def unlock(path,
 
 def party_members(path,
                   zk_hosts,
+                  min_nodes,
```

Comment:
Can `min_nodes` be an optional kwarg with a default set instead of a required arg? It looks like `min_nodes` isn't ever called in this function unless `blocking` is set to `True`, which would make setting `min_nodes` a no-op if `blocking` is still `False`.